RNA_pipeline/RNA_pipeline.wdl

```diff
@@ -0,0 +1,79 @@
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/RNA_pipeline/samtofastq_v1-0_BETA_cfg.wdl" as samtofastq_v1 
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/RNA_pipeline/star_v1-0_BETA_cfg.wdl" as star_v1 
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/RNA_pipeline/rnaseqc2_v1-0_BETA_cfg.wdl" as rnaseqc2_v1 
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/RNA_pipeline/rsem_v1-0_BETA_cfg.wdl" as rsem_v1 
+import "https://raw.githubusercontent.com/broadinstitute/ccle_processing/reorganization/RNA_pipeline/hg38_STAR_fusion.wdl" as hg38_STAR_fusion 
+
+
+workflow RNA_pipeline {
+
+  #samtofastq_v1
+  File input_bam_cram
```

Comment:
what is the reason for explicitly declaring this?

Reply:
It is the base/main input of the workflow  even if used by only one sub-workflow it made sense to me that its namespace would be the one of the main workflow.